packages/rspack/src/loader-runner/index.ts

```diff
@@ -340,36 +340,64 @@ function getCurrentLoader(
 	}
 	return null;
 }
-let tracingCache!: { tracer?: Tracer; activeContext?: Context };
-async function tryTrace(context: JsLoaderContext) {
+
+// FIXME: a temporary fix, we may need to change @rspack/tracing to commonjs really fix it
+let cachedTracing:
+	| {
+			trace: TraceAPI;
+			propagation: PropagationAPI;
+			context: ContextAPI;
+	  }
+	| null
+	| undefined;
+
+async function getCachedTracing() {
 	// disable tracing in non-profile mode
 	if (!process.env.RSPACK_PROFILE) {
-		return {};
+		cachedTracing = null;
+		return cachedTracing;
 	}
-	try {
-		const {
-			trace,
-			propagation,
-			context: tracingContext
-		} = await import("@rspack/tracing");
+	if (cachedTracing) {
+		return cachedTracing;
+	}
+	if (cachedTracing === undefined) {
```

Comment:
this seems has performance issue when user dont install @rspack/tracing，it will try load many times

Reply:
`cachedTracing` will be set to `null` on following calls. Only the first call would match `try catch`

Reply:
cachedTracing will be set to null on following calls. Only the first call would match try catch